src/Symfony/Component/ExpressionLanguage/Parser.php

```diff
@@ -87,34 +90,43 @@ public function __construct(
      * variable 'container' can be used in the expression
      * but the compiled code will use 'this'.
      *
+     * @param int $flags A bit field of `Parser::IGNORE_UNKNOWN_VARIABLES` and `Parser::IGNORE_UNKNOWN_FUNCTIONS`
```

Comment:
`@param int-mask-of<Parser::IGNORE_*>` would be better as type. It would allow psalm and phpstan to know about the fact that it is a bitmask supporting those fields, and they will validate the usages.

This should of course be applied in all methods receiving those flags.

Reply:
Good idea